### ISSUE TITLE
iOS module

### ISSUE THREAD

**Issue by mertoktem** [Shin-NiL/GodotHeyzap#3]
Hello, are you planning to make an ios module?

**Comment by Shin-NiL** [comment]
Hello there.

Unfortunately no, as I don't have any Apple hardware :(

**Comment by mertoktem** [comment]
Can i help you about it? :)

**Comment by Shin-NiL** [comment]
I don't think so, a Mac computer and an iPhone/iPad are needed to develop for iOS and I have none. Maybe someone with the hardware and knowledge can help us sending a pull request. Thanks anyway :)
